tempto-core/src/main/java/com/teradata/tempto/internal/fulfillment/ldap/LdapObjectModuleProvider.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.teradata.tempto.internal.fulfillment.ldap;
+
+import com.google.inject.AbstractModule;
+import com.google.inject.Module;
+import com.google.inject.Scopes;
+import com.teradata.tempto.configuration.Configuration;
+import com.teradata.tempto.initialization.AutoModuleProvider;
+import com.teradata.tempto.initialization.SuiteModuleProvider;
+
+@AutoModuleProvider
```

Comment:
With this in tempto-core jar we will always need `ldap` section in config even if product tests are not using ldap at all.
Othwerwise we will fail on setup.
Short term solution to that is to create separate maven module `tempto-ldap` and only explicitly add dependency to `tempto-ldap.jar` for product tests projects which require that.

Long there I think we should get rid of that `Auto*` stuff and require users to list fulfillers explicitly during tempto initialization.

For now move ldap stuff to separate module.